ui/src/app/views/account-creation/component.html

```diff
@@ -1,52 +1,54 @@
 <div style="padding: 1rem;">
-  <div *ngIf="!accountCreated" class="form-area">
-    <div class="form-section">
-      <label>Given Name:</label>
-      <input type="text" class="input" autofocus [(ngModel)]="profile.givenName">
-    </div>
-    <div class="form-section">
-      <label>Family Name:</label>
-      <input type="text" class="input" [(ngModel)]="profile.familyName">
-    </div>
-    <div class="form-section">
-      <label>Username:</label>
-      <input type="text" class="input" [(ngModel)]="profile.username" (ngModelChange)="usernameChanged()">
-    </div>
-    <div *ngIf="conflictError" class="error">
-      Username is already taken.
-    </div>
-    <div class="form-section">
-      <label>Contact Email:</label>
-      <input type="text" class="input" [(ngModel)]="profile.contactEmail">
-    </div>
-    <div class="form-section">
-      <label>Password:</label>
-      <input type="password" class="input" [(ngModel)]="password">
-    </div>
-    <div class="form-section">
-      <label>Password Again:</label>
-      <input type="password" class="input" [(ngModel)]="passwordAgain">
-    </div>
-    <div class="form-section">
-      <button (click)="createAccount()" class="btn btn-primary">Create Account</button>
-      <span class="spinner spinner-inline" *ngIf="creatingAcccount">
-        Creating account...
-      </span>
-    </div>
-    <div *ngIf="showAllFieldsRequiredError" class="error">
-      All fields are required.
-    </div>
-    <div *ngIf="showPasswordsDoNotMatchError" class="error">
-      Passwords do not match.
-    </div>
-    <div *ngIf="showPasswordLengthError" class="error">
-      Password must be between 8 and 100 characters long.
-    </div>
-    <div style="margin-top: 2rem; text-align: center;">Already have an account?</div>
-    <div style="display: flex; justify-content: space-around;">
-      <button (click)="signIn()" class="btn btn-primary">Sign In</button>
-    </div>
-  </div>
+  <form (ngSubmit)="createAccount()">
+    <div *ngIf="!accountCreated" class="form-area">
```

Comment:
Is this wrapper div still needed? Can you `ngIf` the form instead and move the styling to it as well?

Reply:
the `form-area` style is only on a `div` tag elsewhere in the app, so if I change it here, I'd prefer to make it consistent in the other components as well – thoughts?

Reply:
Yep, that makes sense (assuming those other usages make sense as forms as well), didn't realize these were global styles. Not necessary to make those changes in this PR but feel free to if you'd prefer.